modules/admin/server/routes/admin.server.routes.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var adminPolicy = require('../policies/admin.server.policy'),
+	admin = require('../controllers/admin.server.controller');
+
+module.exports = function(app) {
+	// Users collection routes
+	app.route('/api/users').all(adminPolicy.isAllowed)
```

Comment:
@rhutchison I agree with you, for most cases. For the purposes of an Admin module, it seems that it may need additional consideration. Since there is very specific role requirements, in order to access these routes, it may be better to separate these routes; thus, isolating them from the rest of the api routes. 

I'm not necessarily opposed to this change. Just thought I might express my considerations with these types of api design patterns. 

@lirantal I was trying to convey those concerns. Thanks.